tests/source/brsHamcrest/Test_brsHamcrest_Helpers.brs

```diff
@@ -415,6 +415,63 @@ sub test_coreDoMatch_withItemsApiMethodOverwritten (t as Object)
 end sub
 
 
+
+
+
+sub test_coreDoMatch_targetAndValueAreIdenticalObjects_withDifferentKeyOrder (t as Object)
+    test = setup_brsHamcrest_ObjectMatchers()
+
+    'GIVEN'
+    testTarget = {
+        active: false
+        buttonid: "14_DAYS_CINEMA_TRIAL"
+        buttontext: "Start free trial"
+        category: "CINEMA"
+        colour: "#HEX111"
+        index: 0
+        itemtype: "TRIAL_SUBSCRIPTION"
+        offerid: "C_003088"
+        priceIncreaseMessage: "Movies trial Price increase message"
+        priceIncreasePrice: 20.99
+        priceIncreaseText: "Movies trial price increase text"
+        queued: false
+        sectionnavigation: "MOVIES"
+        statustext: "Start your 14 day free trial"
+        ticketuri: "pkg:/images/account/movies_inactive.png"
+        title: "Sky Cinema 14 Days Trial"
```

Comment:
Please remove brand/commercial mentions (Ideally dummy test data should be used)